doc/design.txt

```diff
@@ -210,15 +210,15 @@ An image trailer has the following structure:
      0                   1                   2                   3
```

Comment:
>> Will the other 32 states reliably trigger a "panic" swap type in the new update code?

> Nope, a weird state should either be NONE or FAIL which end up with a somewhat similar behavior.

Statement like this worry me.

If this is the design document for how the bootloader is supposed to make decisions, and we're still saying that states not listed here (despite the document claiming this is all "possible" states) "should" result in an outcome which is "either [...] NONE or FAIL", which is "somewhat similar behavior", it means that the design has not been completely thought through.